src/main/java/io/strimzi/kafka/quotas/AvailableBytesThrottleFactorPolicy.java

```diff
@@ -23,15 +26,11 @@ public AvailableBytesThrottleFactorPolicy(long availableBytesLimit) {
     }
 
     @Override
-    public double calculateFactor(Collection<VolumeUsage> observedVolumes) {
-        if (shouldThrottle(observedVolumes)) {
-            return 0.0d;
-        } else {
-            return 1.0d;
+    boolean shouldThrottle(VolumeUsage volume) {
+        boolean shouldThrottle = volume.getAvailableBytes() <= availableBytesLimit;
+        if (shouldThrottle) {
+            log.debug("a volume's available bytes of {} is below the {} limit: {}", volume.getAvailableBytes(), availableBytesLimit, volume);
```

Comment:
I think log messages usually begin with a capital letter.

Should we include the log dir and broker id?

Reply:
done, the parameters were in the wrong order too